home.css

```diff
@@ -0,0 +1,811 @@
+body {
+  color: #595c61;
+}
+
+nav {
+  display: none;
+}
+
+#close-button {
+  display: none;
+  position: relative;
+  top: -90%;
+  left: 85%;
+  color: #ec5242;
+}
+
+header {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  background-image: url(./images/background.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 20px 20px 50px 20px;
+}
+
+#header-section {
+  margin-top: 35px;
+}
+
+#menu {
+  color: #595c61;
+}
+
+#salutation {
+  color: #ec5242;
+  font-family: lato, sans-serif;
+  margin: 30px 0 5px 0;
+  font-size: 1.2rem;
+}
+
+#header-section h1 {
+  background: url(./images/h1-background.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-family: 'cocogoose', sans-serif;
+}
+
+#header-section h1 .h1-sub {
+  margin-top: -12px;
+}
+
+#header-section p {
+  border: 3px solid #fff;
+  background-color: #faf7f7;
+  font-size: 0.8rem;
+  font-family: lato, sans-serif;
+  font-weight: 500;
+  padding: 10px;
+  line-height: 1.5;
+  margin: 20px 0 10px 0;
+}
+
+#buy-tickets {
+  display: block;
+  margin-top: 10px;
+  font-size: 1.5rem;
+  font-family: cocogoose, sans-serif;
+}
+
+#date {
+  display: block;
+  margin-top: -8px;
+  font-size: 1.5rem;
+  font-family: cocogoose, sans-serif;
+}
+
+#location {
+  display: block;
+  font-family: lato, sans-serif;
+  font-size: 0.8rem;
+  margin-top: -3px;
+}
+
+#program {
+  background-image: url(./images/program-background.jpg);
+  background-size: cover;
+  background-position: right;
+  color: #fff;
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  padding: 30px 15px;
+}
+
+#program .division-line {
+  width: 50px;
+  height: 2px;
+  background-color: #ec5242;
+  opacity: 0.6;
+  margin: 15px 0 30px 0;
+}
```

Comment:
- [ ] For the program section please use the correct font family to match the given design.
[Hint] you can choose any similar font to the given design from google fonts like `lato` or `roberto` fonts.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/169353087-ffaa1547-40f9-4d3b-aee8-422699ed4c88.png)  | ![image](https://user-images.githubusercontent.com/68796232/169353043-d39ce5e7-0e50-4764-9427-a0f8c617a137.png)  |

Reply:
Thank you for the suggestions. I applied Lato 😄